providers/widget/publish/main.js

```diff
@@ -15,14 +14,18 @@ try {
             `npx provider-cli docs:publish --gh-token ${ghToken}`
         ],
         '🚀Publishing widget 🕹Catalog🕹...'
-    );
+	);
+	
+	utils.execStep(
+		`npm i npm-snapshot -g`
```

Comment:
since you use npx you dont have to install any more

Reply:
ahh i wouldnt know that. That's nice 👍